src/rust/core/server/src/process/builder.rs

```diff
@@ -0,0 +1,161 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+use super::*;
+use crate::threads::*;
+use config::AdminConfig;
+use config::ServerConfig;
+use config::TlsConfig;
+use config::WorkerConfig;
+use entrystore::EntryStore;
+use protocol::{Compose, Execute, Parse};
+use queues::QueuePairs;
+
+const THREAD_PREFIX: &str = "pelikan";
+
```

Comment:
Somewhere in this file, or maybe a standalone readme in this folder would be a good place to add comment blocks on the architecture of the process, including:
- the thread architecture: simply list named threads;
- the communication pattern: what queues are created between which threads, and what type of data get exchanged over those queues;
- the signaling mechanism: the use of waker is not entirely obvious without reading more of the source code, so mentioning that would prepare the reader as they browse further down.

The files under this directory are pretty dense, and we are doing something different from memcached/redis/old pelikan, so it is probably of interest to the readers.